src/main/java/frc/robot/Constants.java

```diff
@@ -29,6 +29,7 @@
   public static final int SPARKMAX_SHOOTER_CAN_ADDR_RIGHT = 5;
   public static final int SPARKMAX_SHOOTER_CAN_ADDR_LEFT = 6;
   public static final int SPARKMAX_LOADER_CAN_ADDR = 7;
+  public static final int SPARKMAX_SHOOTER_HOOD_CAN_ADDR = 8;
   // SparkMAX Encoder Channels
   public static final int SHOOTER_ENCODER_CHAN_A = 1;
   public static final int SHOOTER_ENCODER_CHAN_B = 2;
```

Comment:
I'm not sure what encoder channels are.  I don't think it's what you want.  We should sort this out.